packages/blade/package.json

```diff
@@ -138,6 +138,7 @@
     "use-presence": "1.1.0",
     "@use-gesture/react": "10.2.24",
     "@floating-ui/react": "0.25.4",
+    "react-hot-toast": "2.4.1",
```

Comment:
Update documentation to mention that users need to add this too?

Reply:
Users won't need to install this separately, since it's in `dependencies` 

If users are using react-hot-toast separately in their own codebase then they need to install it.